warehouse/internal/repo/staging.go

```diff
@@ -128,10 +128,15 @@ func (sf *StagingFiles) Insert(ctx context.Context, stagingFile *model.StagingFi
 	if err != nil {
 		return id, fmt.Errorf("marshaling schema: %w", err)
 	}
-
-	bytesPerTableJSON, err := jsonrs.Marshal(stagingFile.BytesPerTable)
```

Comment:
This case was not caught by existing tests since post marshalling value of bytesPerTableJSON was coming as null which was getting saved in the DB. And on unmarshalling this value ("null"), there was no error

Reply:
Default json Marshal behavior, it returns "null" for  `nil`. What we missed here is a test scenario considering an existing entry in warehouse DB?

Reply:
> What we missed here is a test scenario considering an existing entry in warehouse DB?

Yes